projects/flourish-web-sdk-angular/src/lib/enums/language.enum.ts

```diff
@@ -1,5 +1,11 @@
-enum Language {
+export enum Language {
```

Comment:
Same ideia from `environment.enum`

Reply:
I dislike. It's your enum, the customer should translate them environment to your enum and we should have only one option per environment.